app.py

```diff
@@ -13,4 +13,19 @@ def print_board(board):
 
 def get_move():
     position = input("Enter the position for your next move in '#row# #column#' format\n")
-    return list(map(int, list(position.split())))
\ No newline at end of file
+    return list(map(int, list(position.split())))
+
+def alternate_turn():
```

Comment:
probably worth looking for a simple way to alternatrnate/flip a bit in python

Reply:
It seems like `mark` here is serving multiple purposes, you might be conflating the binary `1` or `0` for your logic with the `X` and `O` for your presentation. We should decouple these

Reply:
```
turn = 0

def get_turn(turn_counter):
  # am i on one turn or the other, there are only two
  is_even = turn_counter % 2
  return EVEN if is_even else ODD
  # change to the other turn
  global turn = global turn++ (this is not python.... 🐍 )
```

Reply:
and also maybe increment_turn is its own thing